STYLEGUIDE.md

```diff
@@ -72,3 +76,74 @@ import 'camera.dart';
 import 'game.dart';
 import 'viewport.dart';
 ```
+
+### Identifiers
+
+Use all [effective dart patterns](https://dart.dev/guides/language/effective-dart)
+
+* Avoid using global constants, but if you do use, name it prefixing with a `k`.
+
+```dart
+// Do not:
+const someConstant = 42;
+
+// Do instead:
+const kSomeConstant = 42;
+```
+
+### Comments
+
+* Add an identifier for TODO comments
+* Use fully capitalized TODO notation
```

Comment:
I think we can be more clear adding a "template":

```
// TODO(username): Comment
```

Reply:
done